src/app/Dashboard/AddCard.tsx

```diff
@@ -301,6 +301,8 @@ const PropsConfigForm = ({ onChange, ...props }: PropsConfigFormProps) => {
               onChange={handleNumeric(ctrl.key)}
               onPlus={handleNumericStep(ctrl.key, 1)}
               onMinus={handleNumericStep(ctrl.key, -1)}
+              min={ctrl?.extras?.min}
```

Comment:
There is no need to check undefined on `ctrl`? Just on `extra` is fine I think.

Reply:
Also, is it possible that we can remove a `None` selection or maybe make a field as required? Right now, select any option and then reselect the placeholder really does not do anything...

Reply:
> Also, is it possible that we can remove a `None` selection or maybe make a field as required? Right now, select any option and then reselect the placeholder really does not do anything...

That seems like a good idea. There's a lot we can do to these configs to make the generated form behave better. Probably better for a separate PR.